definitions/features/foreman_proxy.rb

```diff
@@ -5,6 +5,10 @@ class Features::ForemanProxy < ForemanMaintain::Feature
 
   attr_reader :dhcpd_conf_file, :cert_path
 
+  def present?
+    !feature(:foreman_server).present? && find_package('foreman-proxy')
```

Comment:
You can have foreman_proxy feature present also at the same machine as the foreman. This condition probably belongs to the confine block of the self-registered check itself

Reply:
Ah, right. That's correct `present?` is redundant here.